### PR TITLE
Test.build

### PR DESCRIPTION
**Pull request by zsaad**

### CODE REVIEW

File: doc/README/README.environment [hunk 1]
```diff
@@ -4516,3 +4516,10 @@ Variable:  AFNI_ICORR_UBER_USER
 ------------------------------------
 If set to YES, enable distance measures that are not ready for prime
 time in the instacorr interface.
+
+------------------------------------
+Variable:  AFNI_SKIP_TCSV_SCAN
+------------------------------------
+If set to YES, do not scan for CSV files at afni startup. Same as using option -notcsv 
+
+time in the instacorr interface.
```
Comment (mrneont) [inline doc/README/README.environment:4525]: these two lines appear can be deleted?  perhaps a copy+paste leftover?

Comment (zsaad) [reply doc/README/README.environment:4525]: Yup, cut and paste left over.

Comment (zsaad) [reply doc/README/README.environment]: Just pushed a cleaned version

**Comment by mrneont** [comment]
@zsaad : just pushed a @afni-rickr -approved update to the master branch that should help your build here succeed.  Can you pull the master branch into yours here, and see how that proceeds?

**Pull request merged**